README.md

```diff
@@ -90,6 +53,45 @@ Selecting the instances button will also show the individual trace instance.
 This section describes how to take a standard Spring Boot application and prepare it for instrumentation
 using the OpenTracing Java Agent.
 
+### Obtain the OpenTracing Java Agent
+
+This can be done manually, and stored in an appropriate location to be accessed when invoking the service. However for simplicity we have included retrieving the agent as part of the example build:
```

Comment:
Would it be worth adding something like "The code below is incomplete and may be outdated. See the example's `pom.xml` for an actual example."?